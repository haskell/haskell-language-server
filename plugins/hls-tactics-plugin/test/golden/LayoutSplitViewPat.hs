{-# LANGUAGE ViewPatterns #-}

splitLookup :: [(Int, String)] -> String
splitLookup (lookup 5 -> a) = _

