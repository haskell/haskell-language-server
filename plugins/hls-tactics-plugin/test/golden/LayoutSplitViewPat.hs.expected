{-# LANGUAGE ViewPatterns #-}

splitLookup :: [(Int, String)] -> String
splitLookup (lookup 5 -> Nothing) = _
splitLookup (lookup 5 -> (Just s)) = _

